packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.ts

```diff
@@ -352,8 +350,8 @@ export class MetadataHandler {
         itemId: query.sc_itemid,
         language: query.sc_lang,
         // sc_variant is an array in the query params, but we only need the first value
-        variantId: query.sc_variant.split(',')[0],
-        version: query.sc_version,
+        variantId: (query.sc_variant && query.sc_variant.split(',')[0]) || '',
```

Comment:
Should it be _undefined_ by default? If you pass an empty string to GraphQL Editing Layout query, is it working fine?

Reply:
```suggestion
        variantId: query.sc_variant?.split(',')[0],
```